JS/Lesson34/goksel/recipe-app/src/modules/recipes/NewRecipe/NewRecipeForm/newRecipeHelper.ts

```diff
@@ -0,0 +1,192 @@
+
+export const MealCategory = [
```

Comment:
In this particular case, I'd use an enum to store Categories. I think it would be easier to use approach.

Reply:
Probably, you'll need to access all the values of that enum so you can do like this:
```

enum Category {
  Main = 'Main',
  Dessert = 'Dessert',
  Side = 'Side',
}

// But first, you need to use the enum as a type
const allCategories: Category[] = Object.values(Category);
```

Reply:
I used the enum Categories and enum Areas that ı already wrote in models file.